Taskfile.yml

```
@@ -6,6 +6,9 @@ includes:
 vars:
   G_BUILD_DIR: "{{.ROOT_DIR}}/build"
   G_CLP_FFI_JS_BUILD_DIR: "{{.G_BUILD_DIR}}/clp-ffi-js"
+  G_CLP_FFI_JS_CHECKSUM: "{{.G_BUILD_DIR}}/clp-ffi-js.md5"
```

It could be confusing but do we want to deduplicate with
```suggestion
  G_CLP_FFI_JS_BUILD_DIR: "{{.G_BUILD_DIR}}/clp-ffi-js"
  G_CLP_FFI_JS_CHECKSUM: "{{.G_CLP_FFI_JS_BUILD_DIR}}.md5"
```

---

I think let's leave them separate. They happen to have the same prefix but might not always.